index.html

```diff
@@ -1,188 +1,328 @@
+          </li>
+          <li><hr /></li>
+        </ul>
+      </section>
+      <section id="contact-form">
+        <form action="https://formspree.io/f/xoqrddnk" method="POST">
+          <fieldset class="form">
+            <legend>
+              <h2 class="heading">Contact me</h2>
+            </legend>
+            <p class="message">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
             <ul>
-                <li>
-                    <div class="languages">
-                        <p class="language">Languages</p>
-                        <img class="dropdown" src="images2/Union.png" alt="dropdown">
-                    </div>
-                    <ul>
-                        <li>
-                            <div class="ellipse">
-                                <img src="images2/Ellipse1.png"  alt="ellipse">&nbsp;&nbsp;<p class="ellip">JavaSript</p>
-                             </div>
-                        </li>
-                        <li>
-                            <div class="ellipse2">
-                                <img src="images2/Ellipse1(1).png" alt="ellipse1">
-                                <p class="ellip">HTML</p>
-                            </div>
-                        </li>
-                        <li>
-                            <div class="ellipse3">
-                                <img src="images2/Ellipse1(2).png" alt  ="ellipse3">&nbsp;&nbsp;
-                               <p class="ellip">CSS</p>   
-                            </div>
-                        </li>
-                    </ul>
-                </li>
-                <li class="frame">
-                    <p class="skills">Frameworks</p>
-                    <img src="images2/Disabled.png" alt="arrow">
-                </li>
-                <li><hr></li>
-                <li class="frame">
-                    <p class="skills">Skills</p>
-                    <img src="images2/Disabled.png" alt="arrow">
-                </li>
-                <li><hr></li>
+              <li>
+                <input
+                  type="text"
+                  name="user-name"
+                  placeholder="Full Name"
+                  minlength="3"
+                  maxlength="30"
+                  required
+                />
+              </li>
+              <li>
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="example@email.com"
+                  required
+                />
+              </li>
+              <li>
+                <textarea
+                  name="message"
+                  id="message"
+                  cols="5"
+                  rows="10"
+                  placeholder="write your message here"
+                  required
+                ></textarea>
```

Comment:
- [ ] Please make sure that message input has `maxlength="500"` attribute, currently it is not defined 🙌🏻